src/operators.jl

```diff
@@ -746,21 +746,21 @@ julia> collect(edges(h))
 function merge_vertices(g::AbstractGraph, vs)
     labels = collect(1:nv(g))
     # Use lowest value as new vertex id.
-    sort!(vs)
-    nvnew = nv(g) - length(unique(vs)) + 1
+    svs = sort!(vs)
```

Comment:
Since this isn't a ! method, we shouldn't change vs either...
```suggestion
    svs = sort(vs)
```